stock_inventory_valuation_location/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+#. Go to Inventory > Reporting > Inventory Report.
+#. Select Compute (Current Inventory).
+#. Group by location also can use the value measure in pivot view.
```

Comment:
You may either group by location or use the measure value in the pivot view.

Reply:
Change applied. Thanks